ar/src/semantic/bundle.cpp

```diff
@@ -60,7 +62,8 @@ Bundle::Bundle(Context& ctx,
     : _context(ctx),
       _data_layout(std::move(data_layout)),
       _target_triple(std::move(triple)) {
-  ikos_assert_msg(!this->_target_triple.empty(), "empty target triple");
+  // According to the LLVM reference manual the target triple is optional.
+  // ikos_assert_msg(!this->_target_triple.empty(), "empty target triple");
```

Comment:
Let's remove that comment entirely then.

Reply:
Done in next commit.